### PR TITLE
Fix/several fixes

### PR DESCRIPTION
**Pull request by bryanmg**
## Description
- This PR fix several issues with logic, validations or views data.
- Resources affected are: Learning Resources, Reviews and Groups

### CODE REVIEW

File: app/views/students/attendances/index.html.erb [hunk 1]
```diff
@@ -13,26 +13,29 @@
     <div class="level-item has-text-centered">
       <div>
         <p class="heading">Present</p>
-        <p class="title"><%= @attendance_record[true] %></p>
+        <p class="title"><%= @attendance_record[true] ? @attendance_record[true] : 0 %></p>
```
Comment (virgyjimenez) [inline app/views/students/attendances/index.html.erb:16]: isn't this the same as `@attendance_record[true] || 0` ?

File: app/views/students/attendances/index.html.erb [hunk 2]
```diff
@@ -13,26 +13,29 @@
...
-        <p class="title"><%= @attendance_record[false]%></p>
+        <p class="title"><%= @attendance_record[false] ? @attendance_record[false] : 0 %></p>
       </div>
     </div>
     <div class="level-item has-text-centered">
       <div>
         <p class="heading">% of attendance</p>
-        <p class="title"><%= (@attendance_record[true].to_f / @attendance_record.values.sum) * 100 %></p>
+        <p class="title">
+          <%= (@attendance_record[true].to_f / @attendance_record.values.sum * 100).nan? ? 0 : (@attendance_record[true].to_f / @attendance_record.values.sum * 100) %> %
```
Comment (virgyjimenez) [inline app/views/students/attendances/index.html.erb:29]: you're doing the same operation twice, I think there's a better way to do this

File: app/views/students/attendances/index.html.erb [hunk 3]
```diff
@@ -13,26 +13,29 @@
...
         <p class="heading">Present</p>
-        <p class="title"><%= @attendance_record[true] %></p>
+        <p class="title"><%= @attendance_record[true] ? @attendance_record[true] : 0 %></p>
       </div>
     </div>
     <div class="level-item has-text-centered">
       <div>
         <p class="heading">Absent</p>
-        <p class="title"><%= @attendance_record[false]%></p>
+        <p class="title"><%= @attendance_record[false] ? @attendance_record[false] : 0 %></p>
```
Comment (virgyjimenez) [inline app/views/students/attendances/index.html.erb:22]: isn't this the same as `@attendance_record[false] || 0` ?

File: app/views/students/attendances/index.html.erb [hunk 4]
```diff
@@ -13,26 +13,29 @@
...
+        </p>
       </div>
     </div>
     <div class="level-item has-text-centered">
       <div>
         <p class="heading">% of absence</p>
-        <%# TODO: Fix NaN value when there is no attendances fields %>
-        <p class="title"><%= (@attendance_record[false].to_f / @attendance_record.values.sum) * 100 %></p>
+        <p class="title">
+          <%= (@attendance_record[false].to_f / @attendance_record.values.sum * 100).nan? ? 0 : (@attendance_record[false].to_f / @attendance_record.values.sum * 100) %> %
```
Comment (virgyjimenez) [inline app/views/students/attendances/index.html.erb:37]: same as above, you're doing the same operation twice, I think there's a better way to do this

File: app/views/teachers/groups/_form.html.erb [hunk 5]
```diff
@@ -13,30 +13,30 @@
...
     
     <%= form.fields_for :days_of_the_week do |field| %>
-      sunday: <%= field.check_box :sunday? %>
-      monday: <%= field.check_box :monday? %>
-      tuesday: <%= field.check_box :tuesday? %>
-      wednesday: <%= field.check_box :wednesday? %>
-      thursday: <%= field.check_box :thursday? %>
-      friday: <%= field.check_box :friday? %>
-      saturday: <%= field.check_box :saturday? %>
+      sunday: <%= field.check_box :sunday, { checked: group[:days_of_the_week][:sunday] == "1" ? true : false } %>
```
Comment (virgyjimenez) [inline app/views/teachers/groups/_form.html.erb:33]: take a look at how you use `group[:days_of_the_week][:sunday] == "1"`, do you really need a ternary operator?

**Review by virgyjimenez — CHANGES_REQUESTED** [review]

File: app/views/students/attendances/index.html.erb (hunk 2 above)
Comment (bryanmg) [reply app/views/students/attendances/index.html.erb:29]: I apply a helper method to fix this.

File: app/views/students/attendances/index.html.erb (hunk 4 above)
Comment (bryanmg) [reply app/views/students/attendances/index.html.erb:37]: I apply a helper method to fix this.

File: app/views/teachers/groups/_form.html.erb (hunk 5 above)
Comment (bryanmg) [reply app/views/teachers/groups/_form.html.erb:33]: You're right. Fixed.

**Review by virgyjimenez — APPROVED** [review]

**Pull request merged**